griptape/drivers/prompt/huggingface_pipeline_prompt_driver.py

```diff
@@ -34,28 +35,38 @@ class HuggingFacePipelinePromptDriver(BasePromptDriver):
         ),
         kw_only=True,
     )
+    pipe: TextGenerationPipeline = field(
+        default=Factory(
+            lambda self: import_optional_dependency("transformers").pipeline(
+                "text-generation", model=self.model, max_new_tokens=self.max_tokens, tokenizer=self.tokenizer.tokenizer
```

Comment:
Do you know what will happen if you pass model that doesn't support he `text-generation` task?

Reply:
It'll probably break. But part of this refactor allows users to pass in their own `pipe` if they'd like to use a different task type.